{"uuid":"2722a529-dec9-4c38-a884-014cf05a3789","name":"Scenario Outline: The user is presented with error message for invalid code in tryEditor for Max Consecutive Ones","children":["4b186ae2-6e04-4e37-8b09-707854892e60"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"42e56f8e-c1b5-41c4-9faf-855a381abfe6-attachment","type":"image/png"},{"name":"Myscreenshot","source":"736e9270-3132-4d7a-a6ae-8e71d679f765-attachment"}],"parameters":[],"start":1696885544014,"stop":1696885544239},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696885544240,"stop":1696885544240},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696885544241,"stop":1696885544241},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696885544242,"stop":1696885544242}],"start":1696885544008,"stop":1696885544243}